.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -5,28 +5,41 @@ labels: "type-bug"
 ---
 
 <!--
-  If you're new to Python and you're not sure whether what you're experiencing is a bug, the CPython issue tracker is not
-  the right place to seek help. Consider the following options instead:
+  New to Python? The issue tracker isn't the right place to get help.
+  Consider instead:
 
   - reading the Python tutorial: https://docs.python.org/3/tutorial/
-  - posting in the "Users" category on discuss.python.org: https://discuss.python.org/c/users/7
-  - emailing the Python-list mailing list: https://mail.python.org/mailman/listinfo/python-list
-  - searching our issue tracker (https://github.com/python/cpython/issues) to see if
-    your problem has already been reported
+  - posting at https://discuss.python.org/c/users/7
+  - emailing https://mail.python.org/mailman/listinfo/python-list
 -->
 
+## Checklist
+
+<!-- Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker -->
+
+- [ ] I am confident this is a bug in CPython,
+      not a bug in a third-party project
+- [ ] I have searched the CPython issue tracker,
```

Comment:
```suggestion
- [ ] I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
```
Should this be a link?

Reply:
I feel like people who are filing a bug report on the CPython issue tracker probably know where to find the CPython issue tracker ;)

Reply:
Also, I don't think the link would actually render for people reading the template prior to submitting their bug report — they'd just see the raw, unrendered markdown. You'd only see the rendered version if you were reading the bug report on the CPython issue tracker after it had been submitted